### PR TITLE
Modify build scripts, github release tasks and client changes

### PR DESCRIPTION
**Pull request by lahirusamith**
## Purpose
$title

Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/1640

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests

### CODE REVIEW

**Comment by niveathika** [comment]
@lahirusamith Let's not combine independent changes to a single commit. This becomes a hassle in case we want to revert or check history of a specific file

**Comment by lahirusamith** [comment]
> @lahirusamith Let's not combine independent changes to a single commit. This becomes a hassle in case we want to revert or check history of a specific file

I will add separate commits for independent changes. Anyway shall I keep only one PR for merging those commits ?

**Comment by niveathika** [comment]
> > @lahirusamith Let's not combine independent changes to a single commit. This becomes a hassle in case we want to revert or check history of a specific file
> 
> @niveathika I will add separate commits for independent changes. Anyway shall I keep only one PR for merging those commits ?

+1

File: .github/workflows/build-timestamped-master.yml [hunk 1]
```diff
@@ -34,12 +34,5 @@ jobs:
           publishPAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
           ./gradlew publish --scan --no-daemon
-      - name: Archive Error Log
```
Comment (niveathika) [inline .github/workflows/build-timestamped-master.yml:37]: Lets use the latest checkout and setup-java actions

Comment (lahirusamith) [reply .github/workflows/build-timestamped-master.yml:37]: Change added in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/6/commits/51453f0451418aae24c24df75255e7bb9d1c3800

**Review by niveathika — APPROVED** [review]

**Review by daneshk — APPROVED** [review]

**Pull request merged**